libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/FeatureImportances.tsx

```diff
@@ -90,12 +91,14 @@ export class FeatureImportancesTab extends React.PureComponent<
               localization.ModelAssessment.FeatureImportances.GlobalExplanation
             }
           />
-          <PivotItem
-            itemKey={FeatureImportancesTabOptions.LocalExplanation}
-            headerText={
-              localization.ModelAssessment.FeatureImportances.LocalExplanation
-            }
-          />
+          {!ifEnableLargeData(this.context.dataset) ? (
```

Comment:
nit: you can directly render component without ternary operator. 
 {!ifEnableLargeData(this.context.dataset) && (
            <PivotItem
              itemKey={FeatureImportancesTabOptions.LocalExplanation}
              headerText={
                localization.ModelAssessment.FeatureImportances.LocalExplanation
              }
            />
          )}

Reply:
Thanks for pointing this out. Done.